Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -91,7 +91,7 @@ getDriverLoc rideId personId = do
         when (isNothing mbHasReachedNotified && distance <= driverReachedDistance) $ do
           Notify.notifyDriverHasReached personId ride.otp ride.vehicleNumber
           Redis.setExp driverHasReached () 1500
-  return res.currPoint
```

Comment:
Not a backward compatible change. Also mention the purpose of task in description so that dev reviewing can understand.

Reply:
done